test/jdk/ProblemList.txt

```diff
@@ -670,7 +670,6 @@ javax/swing/JFileChooser/6798062/bug6798062.java 8146446 windows-all
 javax/swing/JPopupMenu/4870644/bug4870644.java 8194130 macosx-all,linux-all
 javax/swing/dnd/8139050/NativeErrorsInTableDnD.java 8202765  macosx-all,linux-all
 javax/swing/JEditorPane/6917744/bug6917744.java 8213124 macosx-all
-javax/swing/JRadioButton/4314194/bug4314194.java 8298153 linux-all
```

Comment:
8298153 needs to be closed then if this test is going to be deproblemlisted or add that JBS issue in this PR

Reply:
Added the bug id in test.